include/volk/sse2neon.h

```diff
@@ -0,0 +1,8853 @@
+// Turn off Clang formatting, as 
+// this would make diffs a lot more
```

Comment:
That's great. One more request. Can you add a link to the source of this file? I'd like to make sure that this info doesn't get lost over time.

Reply:
Link has been added. What do you think of the SPDX @jdemel ? should it be MIT or can we use GPL here? how do we handle copied code such as this that comes from a different license?

Reply:
Since this is just a copy from another project and we don't want to modify it, I suggest to stick with [SPDX-License-Identifier: MIT](https://spdx.org/licenses/MIT).